pkg/plugin/vmi_backup_item_action.go

```diff
@@ -79,6 +79,15 @@ func (p *VMIBackupItemAction) Execute(item runtime.Unstructured, backup *v1.Back
 		return nil, nil, errors.WithStack(err)
 	}
 
+	// There's no point in backing up a VMI when it's owned by a VM excluded from the backup
```

Comment:
I changed this to avoid triggering an error (for example, if launcher pod is excluded) if we eventually want to ignore the VMI. I like to have this as a self-contained check before doing anything else.

Reply:
OK